multiversx_sdk_cli/cosign_transaction.py

```diff
@@ -4,7 +4,7 @@
 
 
 class ITransaction(Protocol):
-    guardian_signature: str
+    guardianSignature: str
```

Comment:
OK, the renaming can be applied in `mxpy` (since we use the old `Transaction` class, not the new one from `sdk-core`, for the moment).